pkg/graph/resolvers/resolver_test.go

```diff
@@ -269,7 +272,7 @@ func (s *ResolverSuite) getOrCreateUserAcct(euaUserID string) *authentication.Us
 }
 
 // utility method to get userAcct in resolver tests
-func (s *ResolverSuite) getOrCreateUserAccts(euaUserIDs []string) []*authentication.UserAccount {
+func (s *ResolverSuite) getOrCreateUserAccts(euaUserIDs ...string) []*authentication.UserAccount {
```

Comment:
interesting - guessing this was done to avoid having to write []string{id} everywhere?